hazelcast/src/main/java/com/hazelcast/map/impl/operation/EntryOffloadableSetUnlockOperation.java

```diff
@@ -39,7 +40,7 @@
  * See the javadoc on {@link EntryOperation}
  */
 public class EntryOffloadableSetUnlockOperation extends KeyBasedMapOperation
-        implements BackupAwareOperation, Notifier {
+        implements BackupAwareOperation, Notifier, MutatingOperation {
```

Comment:
Same as others, I believe we shouldn't prevent this from running if quorum is not present, otherwise we end up with a permanently locked entry.

Reply:
fixed